it.emulab.mamedev.clrmamepro/it.emulab.mamedev.clrmamepro.yaml

```diff
@@ -0,0 +1,112 @@
+app-id: it.emulab.mamedev.clrmamepro
+
+runtime: org.winepak.Platform
+runtime-version: 3.0
+sdk: org.winepak.Sdk
+
+command: clrmamepro
+
+add-extensions:
+  org.winepak.Platform.Compat32:
+    directory: lib/32bit
+    version: 3.0
+    add-ld-path: lib
+    no-autodownload: false
+
+tags:
+  - proprietary
+
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --allow=multiarch
+  - --filesystem=host
+
+modules:
+  - name: setup-compat32
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/32bit
+      - ln -s /app/lib/32bit/lib/ld-linux.so.2 /app/lib/ld-linux.so.2
+
+  - name: setup-wine
```

Comment:
This module isn't needed since you're not mounting another Wine extension. This was a mistake in the Notepad++ manifest. My bad, sorry.